### ISSUE TITLE
[New build - DEFAULT] 22w31a: Hotfix another day

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#4813]
Another hotfix build.

Hotfixes:
#4803 ESC key was not working in some cases.
#4808 #4760 #4812 Transparent pipe removal can make next block placed invisible.
#4798 Target selection: can target our ship if it have a shipyard.
#4797 Shipyard IsDocked event does not trigger.

**Comment by tsunamayo** [comment]
Build is up. Thanks

**Comment by AlveKatt** [comment]
Ok, I came back to try again, installed the latest patch, and now all I get is empty space if I try to play. On the bright side, binding the joystick seems to work now. But I can't test it unless I bind it to character movement, because I have nothing to build a ship with.
(Playing via proton in Linux.)

Are there logs anywhere I can share?

**Comment by ProPeach** [comment]
@AlveKatt did you start a new world? The galaxy might have been moved around a little bit since you last played, but if you start a new world you should spawn in the tutorial station

**Comment by AlveKatt** [comment]
Yeah, can only find the build tutorial though. Did you remove the turorial with the little craft?
